### ISSUE TITLE
API gives inconsistent values for gameVersion

### ISSUE THREAD

**Issue by agc93** [beat-saber-modding-group/BeatMods-Website#41]
Querying the `/api/v1/mod` endpoint will give different results for the `gameVersion` property based on whether the query includes an explicit `gameVersion` or not. I'm not sure if this is expected behaviour, but it's pretty confusing at first glance.

To use CustomMenuText as an example:

```text
[~]$ curl -s "https://beatmods.com/api/v1/mod?status=approved" | jq '.[] | select(.name | . == "CustomMenuText") | .gameVersion'
"1.3.0"
"1.6.1"
[~]$ curl -s "https://beatmods.com/api/v1/mod?status=approved&gameVersion=1.8.0" | jq '.[] | select(.name | . == "CustomMenuText") | .gameVersi
on'
"1.8.0"
```

I had a quick look at the code, but I couldn't really keep track of the DAO so sorry I can't pin down where this is being introduced.

**Comment by Ominitay** [comment]
The mod version for `1.6.1` and `1.8.0` is the same. The same happens with BSIPA when the mod versions are the same between game versions. This is probably the reason.

**Comment by agc93** [comment]
But the inconsistency of the results is the major problem, especially given (according to the aliases endpoint) mods are not compatible between 1.6.1 and 1.8.0 so mods will appear as either compatible or not depending on which arguments you include in the query.

**Comment by Ominitay** [comment]
Ofc. Just was pointing out the probable reason. It isn't too big of a problem though, as you can just search for the mod with the `gameVersion`.

**Comment by nike4613** [comment]
Just so you know, this *will not* be fixed (barring a PR from someone), as most of us utterly loathe this codebase and are working on a replacement.

**Comment by agc93** [comment]
Yeah understood on that, and was told much the same a while ago on Discord. Now just waiting for V2!
